koiic/vehicle-store-api
# #1 retrieve all cars in database

**What does this PR do?**

implement the feature to retrieve the list of cars from the database

**Description of Task to be completed?**

As a user, I should be able to fetch the list of cars 

**How should this be manually tested?**

- Fetch the branch with `git fetch origin ft-get-all-cars-01 `
- Check out the branch with `git checkout ft-get-all-cars-01 `
- Run the app: `docker-compose up` or `python manage.py runserver`

**Any background context you want to provide?**

N/A

**What are the relevant pivotal tracker stories?**

N/A

**Screenshots (if appropriate)**

N/A
